### DISCUSSION TITLE
add icon to router list

### DISCUSSION THREAD

**Discussion by MM25Zamanian** [IBM/pwa-lit-template#489] [Ideas]
Hello
I have a suggestion
I'm always upset that the list of routers has limited items
That's why I suggest adding a few items to the router list

1. Label: If the site language is different (make the name unique)
2. Icon: If we use an icon, there is an icon for each root
3. Show in Nav: If there is a menu, should this item be displayed in the menu or not

My suggested format:
```javascript
export const routes: Route[] = [
  {
    show: true,
    path: '/',
    name: 'home',
    icon: 'fa-home',
    label: 'خانه',
    component: 'page-home',
    action: async () => {
      await import('../pages/page-home.js');
    },
  },
  {
    show: true,
    path: '/about',
    name: 'about',
    icon: 'fa-info',
    label: 'درباره ما',
    component: 'page-about',
    action: async () => {
      await import('../pages/page-about.js');
    },
  },
  {
    show: false,
    path: '(.*)',
    name: 'not-found',
    component: 'page-not-found',
    action: async () => {
      await import('../pages/page-not-found.js');
    },
  },
];
```

**Comment by MM25Zamanian** [comment]
If you agree with this idea
Let me know so I can start presenting a new `interface`

**Comment by abdonrd** [comment]
For now we want to keep the template as simple as possible.

We leave those kinds of extra changes to the owner of the application.

**Reply by MM25Zamanian** [reply]
If you make the changes optionally, you will surely get good feedback because beginners with a simple template but with additional features are more comfortable.
